locale/en/partner_descriptions.json

```diff
@@ -140,10 +140,10 @@
 	"160_short_description": "<strong>The Wall Street Journal</strong> (WSJ) is an American newspaper, with a focus on business and finance. This collection gives access to the Chinese (Simplified) edition of WSJ for 3 years.",
 	"161_short_description": "<strong>The Wall Street Journal</strong> (WSJ) is an American newspaper, with a focus on business and finance. This collection gives access to the Chinese (Traditional) edition of WSJ for 3 years.",
 	"162_short_description": "<strong>Hindu Tamil Thisai</strong> (colloquially known as The Hindu Tamil) is a Tamil daily newspaper headquartered at Chennai. It is published by The Hindu Group.",
-	"163_short_description": "<strong>The Indian Express</strong> is an English-language Indian daily newspaper founded in 1932.  is published in Mumbai by the Indian Express Group.",
+	"163_short_description": "<strong>The Indian Express</strong> is an English-language Indian daily newspaper founded in 1932. It is published in Mumbai by the Indian Express Group.",
 	"164_short_description": "<strong>Arcanum</strong> is the largest and continuously expanding digital periodical database from Eastern Europe, which contains scientific and specialized journals, encyclopaedias, weekly and daily newspapers and more",
-	"165_short_description": "<strong>Les Jours</strong> is an independent and generalist online media based in France. Les Jours specialises in long-form reporting on topics such as media moguls, feminism, pollution, discrimination etc.",
-	"166_short_description": "<strong>East View Press</strong> publishes periodicals and books focused on international and public policy issues, particularly related to the former Soviet Union and China. Their publications cover topics like foreign policy, international security, Slavic and Sino studies, and geography, with content translated and edited to make global research accessible in English​.",
+	"165_short_description": "<strong>Les Jours</strong> is an independent and generalist online media based in France. Les Jours specialises in long-form reporting on topics such as media moguls, feminism, pollution, discrimination, etc.",
+	"166_short_description": "<strong>East View Press</strong> publishes periodicals and books focused on international and public policy issues, particularly related to the former Soviet Union and China. Their publications cover topics like foreign policy, international security, Slavic and Sino studies, and geography, with content translated and edited to make global research accessible in English.",
```

Comment:
The change on this line is practically impossible to see in GitHub. There was a zero-width space (Unicode 200B) character between "English" and the last full stop. I noticed it because vim highlights such characters. I don't know why it was there, but I'm quite sure that it's unnecessary.